reconcile/utils/saasherder.py

```diff
@@ -178,6 +184,24 @@ def _validate_saas_files(self):
             for duplicate in duplicates:
                 logging.error(msg.format(duplicate))
 
+    def _check_saas_file_env_combo_unique(self, saas_file_name, env_name):
+        # max tekton pipelinerun name length can be 63.
+        # leaving 12 for the timestamp leaves us with 51
+        # to create a unique pipelinerun name
+        tkn_long_name = f"{saas_file_name}-{env_name}"
+        tkn_name = tkn_long_name[:50]
```

Comment:
Make this 50 a constant, please. Magic numbers end up biting us.

Reply:
done